_posts/blog/2018-09-10/2018-09-10-the-beginners-guide-to-decentralization.markdown

```diff
@@ -0,0 +1,97 @@
+
+This image shows what a centralized system looks like. The participants in the system are on the outside, and there is one person or organization in the middle that everyone else is dependent upon.
+
+![Visual model of a centralized network](Centralized-network.png "Visual model of a centralized network")
+*Centralized network with a central point of control*
+
+Centralization has its benefits. It removes ambiguity about who is in control and allows for swift action to remedy problems. For computer networks, it’s much easier to build and maintain centralized systems than decentralized ones. Having a strong leader controlling a company can help keep everyone focused.
+
+Centralization also has drawbacks. There may not be ambiguity about who is in control, but that doesn’t mean those in control are actually competent, or moral. Centralized systems are easier to build technically, but they are also more fragile, since they have a single point of failure (if the person / organization in charge is gone, so is the system). Being permissioned means that those in power might exclude people they don’t like or agree with.
+
+Centralized systems can change based on who is in control. Even if the system is run well by competent and moral people today, it could be run poorly by different people tomorrow.
+
+There is an alternative to having systems so tightly controlled: Decentralization.
+
+### What Are Decentralized Systems?
+
+As we’ve said, _decentralized_ means that there is no single person or organization that has control over everyone else in a given system.
+
+Participants in decentralized systems have the ability to make decisions themselves and aren’t forced to rely on anyone who controls the system. Let’s look at two examples.
+
+> **A decentralized political system** is one where no person or organization has the ability to govern everyone else. A completely decentralized political system would be an anarchy (absence of a governing structure). Modern democratic systems have some elements of decentralization, where participants in the system do have some control over the decision-making process and there are multiple organizations which exist to ensure power doesn’t become centralized within one portion of the government.
+
+> **A decentralized computer network** is a system where the computers talk directly to each other and there is no central point of control. Most online platforms today work by having a lot of computers called servers that their users are forced to connect to if they want to use the service. They are centralized and completely control the system. A decentralized computer network has no central servers that control access to the network; each users’ computer connects directly to other users’ computers, making a network of computers that no one controls. OpenBazaar is such a network, as are networks like Bitcoin and Bittorrent.
```

Comment:
I think you might want a colon instead of a semi-colon, since the second half of the sentence can't stand alone, and is an explanation of the first half.